plugins/inputs/slurm/slurm.go

```diff
@@ -0,0 +1,321 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package slurm
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	goslurm "github.com/pcolladosoto/goslurm/v0038"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type Slurm struct {
+	URL      string `toml:"url"`
+	Username string `toml:"username"`
+	Token    string `toml:"token"`
+
+	IgnoreDiag         bool `toml:"ignoreDiag"`
+	IgnoreJobs         bool `toml:"ignoreJobs"`
+	IgnoreNodes        bool `toml:"ignoreNodes"`
+	IgnorePartitions   bool `toml:"ignorePartitions"`
+	IgnoreReservations bool `toml:"ignoreReservations"`
+
+	ResponseTimeout config.Duration `toml:"response_timeout"`
```

Comment:
Can you group all the exported variables together please

Reply:
That should have been done on 2b0814c47b607ec0009e313358aefa2faa798189 😜

Reply:
That should have been done on 2b0814c47b607ec0009e313358aefa2faa798189 😜